src/constants/axis_placeable_objects.js

```diff
@@ -0,0 +1,30 @@
+/**
+* Copyright 2012-2020, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+module.exports = {
+    axisRefDescription: function (axisname,lower,upper) {
+        return [
+            `If set to a ${axisname} axis id (e.g. *${axisname}* or`,
```

Comment:
Nice abstraction! But again we need to downgrade to ES5 - ie just concatenate strings, no backtick templates.

Reply:
:+